src/modules/ekf2/EKF2.cpp

```diff
@@ -1229,7 +1230,8 @@ void EKF2::PublishStatusFlags(const hrt_abstime &timestamp)
 		status_flags.cs_ev_vel                = _ekf.control_status_flags().ev_vel;
 		status_flags.cs_synthetic_mag_z       = _ekf.control_status_flags().synthetic_mag_z;
 		status_flags.cs_vehicle_at_rest       = _ekf.control_status_flags().vehicle_at_rest;
-		status_flags.cs_gps_yaw_fault         = _ekf.control_status_flags().gps_yaw_fault;
```

Comment:
I agree that the `gps_yaw_fault` flag should go in the fault status flags, but you shouldn't just remove it :D

Reply:
Thanks, fixed.